### ISSUE TITLE
Combining the setAutoFilter and setAutoSize

### ISSUE THREAD

**Issue by cwolcott** [PHPOffice/PhpSpreadsheet#2274]
This is:

```
- [X] a bug report
- [ ] a feature request
```

### What is the expected behavior?
I would have hoped that using the **setAutoSize** function would have appropriately determined the correct size for a column when the function **setAutoFilter** had already be applied.
```php
        Font::setTrueTypeFontPath("/usr/share/fonts/msttcore/");
        Font::setAutoSizeMethod(Font::AUTOSIZE_METHOD_EXACT);
        ...
        // Apply filtering capability to the worksheet
        $worksheet->setAutoFilter($worksheet->calculateWorksheetDimension());

        // Apply auto size to all columns
        foreach (range('A',$workSheetMaxColumns) as $col) { $worksheet->getColumnDimension($col)->setAutoSize(true); }
        ...
```

### What is the current behavior?
It does not seem to take into account the filter icon shown in the column.  See attached images
<img width="295" alt="AutoFilter AutoSize NotCalculated" src="https://user-images.githubusercontent.com/3777526/129465820-a7cd6ab7-4547-410a-9969-8483aa58d3c6.png">

When I double click on the column seperator in excel it widens the column widths to my satisfaction.
<img width="408" alt="AutoFilter AutoSize ExcelCalculated" src="https://user-images.githubusercontent.com/3777526/129465842-1b7a7716-ce65-47d4-8800-e84a05c749b4.png">

### Which versions of PhpSpreadsheet and PHP are affected?
version 1.18.0 and master branch

**Comment by cwolcott** [comment]
This has been fixed in issue #2273

**Comment by cwolcott** [comment]
I thought this was fixed, but I guess I fixed it myself and then lost the fix when upgrading from 1.18.0 to 1.20.0.  I am trying to figure out what I did to fix it.

Before in 1.18.0 with may fix:
![image](https://user-images.githubusercontent.com/3777526/143259301-a85fb08f-9c9c-4faf-93b0-7cdc7e88191d.png)

After in 1.20.0 after losing my fix:
![image](https://user-images.githubusercontent.com/3777526/143259448-c95ea54e-e016-44c0-8712-fac0f922269c.png)
